django_project/geosight/data/api/v1/dashboard.py

```diff
@@ -84,3 +85,26 @@ def destroy(self, request, slug=None):
     def delete(self, request, slug=None):
         """Destroy an object."""
         return super().delete(request, slug=slug)
+
+
+class DashboardGroupList(BaseApiV1ResourceReadOnly):
```

Comment:
I think instead creating new API, you could use additional action to DashboardViewSet
example: http://localhost:2000/api/v1/dashboards/groups/

    @action(detail=False, methods=['get'])
    def groups(self, request):
        querysets = self.get_queryset()
        groups = querysets.values_list(
            'group__name', flat=True
        ).distinct().order_by('group__name')
        return Response(groups)

Reply:
Updated